experiment/simulation/index.html

```diff
@@ -1,16 +1,133 @@
-<!DOCTYPE html>
-<html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <!--link rel="stylesheet" href="./css/main.css">-->
-</head>
-<body>
-		<!-- Your code goes here-->
-	
-	<p><li>This simulation has some <a href="../../../Introduction.html" target="PARENT">software dependencies</a>. We highly recommend you to use our customized <a href="https://vlead.vlabs.ac.in/development/#virtual-box" target="blank">VirtualBox</a> for the smooth running of these simulations.</li></p>
-	<p>Click <a class="qtlink" href="exp1.html" onclick="window.open('./exp1.html', null,'scrollbars=yes, width=' + (screen.availWidth) +',height=' + (screen.availHeight)); return false;"> here	</a> to perform the experiment. </p>
-
-	<!-- Add JS at the bottom of HTML file --> 
-        <script src="./js/main.js"></script>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<title>Document</title>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.7-b/css/style.min.css">
+	<link rel="stylesheet" href="css/main.css">
+	<!-- <script src="https://cdn.jsdelivr.net/npm/interactjs/dist/interact.min.js"></script> -->
+</head>
+
+<body>
+	<div class="container" style="margin-top: 1%;">
+		<div class="columns is-centered">
+			<div class="column is-one-quarter v-instruction-title">
+				Tasks
+			</div>
+			<div class="column">
+				<div class="v-tabs">
+					<ul>
+						<li class="is-active" id="Task1" onclick="changeTabs(event)">
+							<a>
+								Task 1
+							</a>
+						</li>
+						<li id="Task2" onclick="changeTabs(event)">
+							<a>
+								Task 2
+							</a>
+						</li>
+						<li id="Task3" onclick="changeTabs(event)">
+							<a>
+								Task 3
+							</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+
+		<div class="columns is-centered">
+			<div class="column">
+				<div class="v-instruction-container">
+					<div class="v-instruction-title" id="TaskTitle">Half Adder</div>
+					<div class="v-instruction-text" id="TaskDescription">
+						Implement a 1-bit half adder
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>

```

Comment:
Container class ends here. Are the elements below this line outside of the container div?

Reply:
Yes, the container class from virtual styles actually adds large left and right padding to the page which makes the content difficult to fit in. So to fit everything in the screen, I took it outside the container.